.github/workflows/RollPyTorch.yml

```diff
@@ -106,8 +106,28 @@ jobs:
         git fetch --recurse-submodules=no
         git checkout main
         git pull origin main
+        git checkout -b rollpytorch/${{ env.PT_RELEASE }}
         git add pytorch-hash.txt pytorch-requirements.txt torchvision-requirements.txt lib/Dialect/Torch/Transforms/AbstractInterpLibrary.cpp include/torch-mlir/Dialect/Torch/IR/GeneratedTorchOps.td
-        git diff --cached --exit-code || (git commit -m "update PyTorch version to ${{ env.PT_RELEASE }}" && git push --set-upstream origin main)
+        git diff --cached --exit-code || git commit -m "update PyTorch version to ${{ env.PT_RELEASE }}"
+
+    - name: Create pull request
+      uses: peter-evans/create-pull-request@v5.0.1
+      with:
+        author: Roll PyTorch Action <torch-mlir@users.noreply.github.com>
+        branch: rollpytorch/${{ env.PT_RELEASE }}
+        body: |
+          torch version: ${{ env.PT_RELEASE }}
+          torch commit hash: ${{ env.PT_HASH }}
+          torchvision version: ${{ env.PTVISION_RELEASE }}
+        commit-message: |
+          update PyTorch version to ${{ env.PT_RELEASE }}
+
+           - torch version: ${{ env.PT_RELEASE }}
+           - torch commit hash: ${{ env.PT_HASH }}
+           - torchvision version: ${{ env.PTVISION_RELEASE }}
+        committer: Roll PyTorch Action <torch-mlir@users.noreply.github.com>
+        title: update PyTorch version to ${{ env.PT_RELEASE }}
+        reviewers: ashay, powderluv, vivekkhandelwal1
```

Comment:
this is awesome. Do you think we can mark as reviewed to auto-merge if all tests pass ?

Reply:
That's a nice idea!  I believe we can have another workflow that listens to newly created PRs; then if the PR is from the RollPyTorch action updating a restricted set of files, then the workflow could use the GitHub CLI to approve ([`gh pr review --aprove`](https://cli.github.com/manual/gh_pr_review)) and auto-merge ([`gh pr merge --auto`](https://cli.github.com/manual/gh_pr_merge)) the PR pending status checks.

I'd prefer to do that in stages, though.  I'm afraid of making too many changes in one PR, so if you're okay with it, I'd like to land this first before adding the auto-approving / auto-merging logic.